### PR TITLE
Improves the limb check for the pun

### LINKED ISSUE #19912: Showing up as all right when having all limbs
right
@optimumtact

### PR DESCRIPTION
**Pull request by optimumtact**
Fixes #19912

### CODE REVIEW

**Comment by kevinz000** [comment]
why

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -203,12 +203,19 @@
...
 			msg += "<B>[t_He] [t_has] \a \icon[I] [I] embedded in [t_his] [BP.name]!</B>\n"
 
+	//stores how many left limbs are missing
+	var/l_limbs_missing = 0
 	for(var/t in missing)
 		if(t=="head")
 			msg += "<span class='deadsay'><B>[capitalize(t_his)] [parse_zone(t)] is missing!</B><span class='warning'>\n"
 			continue
+		if(t == "l_arm" || t == "l_leg")
+			l_limbs_missing += 1
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/examine.dm]: ```++```

**Comment by optimumtact** [comment]
It was broken so I fixed it.

**Comment by lzimann** [comment]
you are all right oranges

File: code/modules/mob/living/carbon/human/examine.dm [hunk 2]
```diff
@@ -203,12 +203,19 @@
 		for(var/obj/item/I in BP.embedded_objects)
 			msg += "<B>[t_He] [t_has] \a \icon[I] [I] embedded in [t_his] [BP.name]!</B>\n"
 
+	//stores how many left limbs are missing
+	var/l_limbs_missing = 0
 	for(var/t in missing)
 		if(t=="head")
 			msg += "<span class='deadsay'><B>[capitalize(t_his)] [parse_zone(t)] is missing!</B><span class='warning'>\n"
 			continue
+		if(t == "l_arm" || t == "l_leg")
```
Comment (AnturK) [inline code/modules/mob/living/carbon/human/examine.dm]: These don't have defines ?

Comment (optimumtact) [inline code/modules/mob/living/carbon/human/examine.dm]: yeah, that's what tripped me up the first time as I had assumed someone had added defines for them, but it was not to be.

**Comment by AnturK** [comment]
It's not all right now.

**Pull request merged**